### PR TITLE
Feat: support vela def vet the input "dir" and many cue files 

### LINKED ISSUE #4744: [Feature] "vela def vet" can support  the input "dir" and many cue files
Now "vela def vet" only can vet on cue file, it is necessay to support the specified dir, the local dir and many cue files, such as:
- vela def vet /home/dir/*
- vela def vet dir/*
- vela def vet *
- vela def vet a.cue b.cue c.cue

And output the vet results of all the input cue files.

### PR DESCRIPTION
**Pull request by Mcduller**
### Description of your changes

Fixes #4744

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: references/cli/def.go [hunk 1]
```diff
@@ -1033,35 +1032,53 @@ func NewDefinitionDelCommand(c common.Args) *cobra.Command {
...
-			}
-			def := pkgdef.Definition{Unstructured: unstructured.Unstructured{}}
-			config, err := c.GetConfig()
-			if err != nil {
-				return err
-			}
-			if err := def.FromCUEString(string(cueBytes), config); err != nil {
-				return errors.Wrapf(err, "failed to parse CUE")
+			for _, arg := range args {
+				files, err := utils.LoadDataFromPath(context.Background(), arg, utils.IsCUEFile)
```
Comment (Somefive) [inline references/cli/def.go:1052]: context.Background() -> cmd.Context()

File: references/cli/def.go [hunk 2]
```diff
@@ -1033,35 +1032,53 @@ func NewDefinitionDelCommand(c common.Args) *cobra.Command {
...
+				if err != nil {
+					return errors.Wrapf(err, "failed to get file from %s", arg)
+				}
+				for _, file := range files {
+					validateRes, err := validateSingleCueFile(file.Path, file.Data, c)
+					if err != nil {
+						return err
+					}
+					streams.Infonln(validateRes)
```
Comment (Somefive) [inline references/cli/def.go:1061]: We can use fmt.Fprintf(cmd.Stdout, "xxx") instead of streams.Infoln

**Review by Somefive — COMMENTED** [review]
Generally LGTM after nit, great job!

File: references/cli/def.go (hunk 1 above)
Comment (Mcduller) [reply references/cli/def.go:1052]: ok

File: references/cli/def.go [hunk 3]
```diff
@@ -1033,35 +1032,53 @@ func NewDefinitionDelCommand(c common.Args) *cobra.Command {
...
-			"> vela def vet my-def.cue",
-		Args: cobra.ExactArgs(1),
+			"> vela def vet my-def.cue\n" +
+			"# Command below will validate all the cue file\n" +
+			"> vela def vet my-def1.cue my-def2.cue my-def3.cue\n" +
+			"# Command below will validate all the cue in specific dir\n" +
+			"> vela def vet ./test/" +
+			"# Command below will validate all the cue files in all specific dirs" +
+			"> vela def vet ./test1/ ./test2/",
+		Args: cobra.MinimumNArgs(1),
```
Comment (charlie0129) [inline references/cli/def.go:1049]:
```suggestion
			"> vela def vet my-def.cue\n" +
			"# Validate every cue file provided\n" +
			"> vela def vet my-def1.cue my-def2.cue my-def3.cue\n" +
			"# Validate every cue file in the specified directories" +
			"> vela def vet ./test1/ ./test2/",
		Args: cobra.MinimumNArgs(1),
```

**Review by charlie0129 — COMMENTED** [review]
Minor usage text improvements

File: references/cli/def.go (hunk 3 above)
Comment (Mcduller) [reply references/cli/def.go:1049]: ok

**Review by Somefive — APPROVED** [review]
Great job! Thanks for the contribution!

**Pull request merged**